src/components/learner-credit-management/BudgetDetailTabsAndRoutes.jsx

```diff
@@ -76,29 +104,43 @@ const BudgetDetailTabsAndRoutes = ({
     );
   };
 
+  const [refreshMembersTab, setRefreshMembersTab] = useState(false);
```

Comment:
Having to prop-drill this refresh flag into multiple components isn't the most scalable solution, IMO.  In circumstances where updates need to be synchronized across many components, I've found that context-based solutions generally work better.